docs/tools/update-git-versions.json.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+git_versions_file=$1
+
+if [[ -z $git_versions_file ]]
+then
+	echo "Usage: update-git-versions.json.sh <git-versions.json file> [--in-place]"
+	exit 1
+fi
+
+# Clone a fresh repository
+if [[ ! -d git ]]
+then
+	git clone https://github.com/git/git --quiet
+else
+	git -C git fetch --quiet
+	git -C git reset --hard origin/master --quiet
+fi
+
+# “taggerdate:iso-local” forces an ISO timestamp in the local time zone,
+# which we override with UTC in order to reliably extract timestamps.
+# “refname:short” displays tags as “v1.2.3” and not “refs/tags/v1.2.3”.
+# Versions earlier than 2 are excluded, because they are outdated
+# anyway by now. Release candidates are also removed.
+TZ=UTC git -C git tag -l 'v*' --format='%(refname:short)	%(taggerdate:iso-local)' --sort=-taggerdate \
+	| grep -v '^v[01]' \
+	| grep -v 'rc' \
+	| perl -pe 's/(\d{4,}-\d{2,}-\d{2,}) \d{2,}:\d{2,}:\d{2,} \+0000/\1/g' \
+	| cut -c 2- \
+	> git-releases.tsv
+
+# Translate the TSV file to JSON.
+jq --slurp --raw-input --raw-output --tab \
```

Comment:
Minor nit: `jq` is not available everywhere. You could check if it is installed like this: https://github.com/Autodesk/enterprise-config-for-git/blob/4d592a7f18dd6328f278ca8cdf1f6a81c7411fb1/lib/setup_helpers.sh#L48-L50

Reply:
True, but this script is not meant to be invoked by anyone but us maintainers. As this script just helps keep the release information up-to-date, the vulnerabilities still need to be added manually. Hence, I guess it’s safe to not do thorough checking in this script :slightly_smiling_face:.